modules/admission-webhook-types.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-types_{context}"]
+= Types of admission webhooks
+
+Cluster administrators can include _mutating admission webhooks_ or _validating admission webhooks_
+in the admission chain of the API server.
+
+[id="mutating-admission-webhooks_{context}"]
+*Mutating admission webhooks* are invoked during the mutation phase of the admission process, which allows modification of the resource content before it is persisted. One example of a mutating admission webhook is the Pod Node Selector Pod Node Selector feature, which uses an annotation on a namespace to find a label selector and add it to the pod specification.
+
+[id="mutating-admission-webhooks-sample"]
+.Sample mutating admission webhook configuration
+
+[source,yaml]
+----
+- apiVersion: admissionregistration.k8s.io/v1beta1
+  kind: MutatingWebhookConfiguration <1>
+  metadata:
+    name: <controller_name> <2>
+  webhooks:
+  - name: <webhook_name> <3>
+    clientConfig: <4>
+      service:
+        namespace:  <5>
```

Comment:
This `namespace` field and `name` don't have any values set, should they? The example for the validating admission webhook does.

Reply:
Well spotted. I have applied the same values here as in the validating admission webhook example.